spoon-client/src/main/java/com/squareup/spoon/Spoon.java

```diff
@@ -249,7 +251,7 @@ static StackTraceElement findTestClassTraceElement(StackTraceElement[] trace) {
 
   private static StackTraceElement extractStackElement(StackTraceElement[] trace, int i) {
     //Stacktrace length changed in M
-    int testClassTraceIndex = Build.VERSION.SDK_INT >= 23 ? (i - 2) : (i - 3);
+    int testClassTraceIndex = Build.VERSION.SDK_INT >= MARSHMALLOW_API_LEVEL ? (i - 2) : (i - 3);
```

Comment:
Could've use `Build.VERSION_CODES` but it requires additional dependency in this module.